docs/product/relay/projects.mdx

```diff
@@ -108,3 +108,17 @@ An empty list rejects all origins. Use the default `["*"]` to allow all origins.
   }
 }
 ```
+
+`config.features`
+
+Enable product features that are not yet enabled by default.
+
+```json
+{
+  "config": {
+    "features": ["organizations:session-replay"]
+  }
+}
+```
+
+A full list of features can be found [here](https://github.com/getsentry/relay/blob/master/relay-dynamic-config/src/feature.rs).
```

Comment:
Since we now have features documented in relay docs:
```suggestion
A full list of features can be found [here](https://getsentry.github.io/relay/relay_dynamic_config/enum.Feature.html#variants).
```